packages/certificate-summary/src/index.ts

```diff
@@ -18,8 +18,24 @@ export interface CertificateSummary {
   purchaseDeliveryPosition?: string;
   purchaseOrderNumber?: string;
   purchaseOrderPosition?: string;
+  orderQuantityInKG?: string;
```

Comment:
Any reason why `orderQuantityInKG` is a string ?
I'm not really in favor of that property name and the fact that this is a string. I would rather see an object with value and unit like in the certificates schemas. Then we can still (try to) make the interpretation in the code using this lib, otherwise prepare for a good time of unit conversion to always end up with kg.

Reply:
I made `orderQuantityInKG` a string for consistency, as if you look at the interface `CertificateSummary`, all the properties, including numbers, are strings. I thought it would make it easier to parse later, but I can happily change it back to a number as it will simplify some of the logic.

As for using an object, based on the fixtures and the previous all hands meeting, I made the assumption that the unit would always be kilograms, which I see now may not be the case.